tests/mo-timing-synchronization/OPS/content_001.xhtml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+        <meta charset="utf-8" />
+		<title>Basic Media Overlay Test</title>
```

Comment:
Nit: sentence case for titles (in our tests). Applies throughout.

Reply:
+1